symfony/framework-bundle/4.2/public/index.php

```diff
@@ -9,7 +9,11 @@
 if ($_SERVER['APP_DEBUG']) {
     umask(0000);
 
-    Debug::enable();
+    if (class_exists(Debug::class)) {
+        Debug::enable();
```

Comment:
I understand wrapping this in an if and only running when it exists, but I don’t understand throwing an exception if it doesn’t exist.